lib/utils/stream.js

```diff
@@ -92,11 +102,18 @@ var Stream = function() {
 Stream.prototype.pipe = function(destination) {
   this.on('data', function(data) {
     destination.push(data);
-  });
+  }, destination);
 
   this.on('done', function(flushSource) {
     destination.flush(flushSource);
-  });
+  }, destination);
+
+  return destination;
+};
+
+Stream.prototype.unpipe = function(destination) {
```

Comment:
if we can just throw away the pipeline as David suggested, than we can probably get rid of unpipe